src/instructlab/model/train.py

```diff
@@ -279,9 +303,67 @@
 @click.option(
     "--legacy",
     is_flag=True,
-    default=False,
     help="if true, enables the legacy linux training codepath from release 0.17.0 and prior.",
 )
+@click.option(
+    "--strategy",
+    type=click.Choice(["lab-multiphase"], case_sensitive=False),
+    show_default=True,
+    help="If chosen, will run the selected training strategy instead of a single training run.",
+)
+@click.option(
+    "--phased-base-dir",
+    type=clickpath_setup(is_dir=True),
+    default=lambda: DEFAULTS.PHASED_DIR,
+    help="Base directory for organization of end-to-end intermediate outputs.",
+)
+@click.option(
+    "--phased-phase1-data",
```

Comment:
(Thought for the future) The number of phases and types of inputs seems tightly coupled to a particular strategy. Other strategies may have different inputs and/or number of steps. Eventually, we may have to move these configurations from CLI to input files (e.g. have a `config.json` for a training strategy input directory that is dynamically loaded and injected into the strategy runner.)

Reply:
100% agree. enumerating these strategy knobs/levers would be a lot- I'm going to fix this in a subsequent PR